include/mptcpd/plugin.h

```diff
@@ -166,20 +232,34 @@ struct mptcpd_plugin_ops
                                  struct sockaddr const *raddr,
                                  bool backup,
                                  struct mptcpd_pm *pm);
+
+        /**
+         * @brief Network monitor event handlers.
+         *
+         * Set of network monitoring event handling functions provided
+         * by the path manager plugin.
+         */
+        struct mptcpd_plugin_nm_ops const *nm_ops;
```

Comment:
The goal here is to separate the path management event handlers from the network monitoring event handlers in the plugin interface.  However, it's a bit odd to add this level of indirection for the network monitoring event handlers but not the path management event handlers above.  Should we leave this `struct` as is, or move the path management event handlers to a separate `mptcpd_plugin_pm_ops` `struct`, as we do for `nm_ops`?  We could also drop  `struct mptcpd_plugin_nm_ops` entirely, and add the event handlers as direct members of `struct mptcpd_plugin_ops`.